reana_client/version.py

```diff
@@ -14,4 +14,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = "0.6.0.dev20190912"
+__version__ = "0.6.0"
```

Comment:
Please assemble some changes to `CHANGES.rst` using the style from v0.5.0 and I'l then reformulate while merging...

Reply:
sorry, I forgot to add it to the commit.